src/platform/datapath_raw.c

```diff
@@ -434,7 +436,8 @@ CxPlatSocketSend(
         (uint16_t)SendData->Buffer.Length,
         CASTED_CLOG_BYTEARRAY(sizeof(*RemoteAddress), RemoteAddress),
         CASTED_CLOG_BYTEARRAY(sizeof(*LocalAddress), LocalAddress));
-    CxPlatFramingWriteHeaders(Socket, LocalAddress, RemoteAddress, &SendData->Buffer);
+    CxPlatFramingWriteHeaders(
```

Comment:
There should be no need to pass family here, since the addresses are already passed in.

Reply:
oh right, brainfart.

Reply:
Ok removed now